src/sentry/integrations/github/utils.py

```diff
@@ -18,7 +18,8 @@ def get_jwt():
         # JWT expiration time (10 minute maximum)
         'exp': exp,
         # Integration's GitHub identifier
-        'iss': options.get('github-app.id'),
+        'iss': github_id or options.get('github-app.id'),
     }
 
-    return jwt.encode(payload, options.get('github-app.private-key'), algorithm='RS256')
+    return jwt.encode(payload, github_private_key or options.get(
```

Comment:
nit: split it into two lines

```py
private_key = github_private_key or options.get('github-app.private-key')
return jwt.encode(payload, private_key, algorithm='RS256')
```

Actually this whole method I would write this at the top

```py
def get_jwt(github_id=None, github_private_key=None):
    if github_id is None:
        github_id = options.get('github-app.id')
    if github_private_key is None:
        github_private_key = options.get('github-app.private-key')

    # same code follows but use these two variables.
```

Reply:
this works 👌